docs/how-to/contribute.md

```diff
@@ -132,52 +133,25 @@ fixes to other branches.
 (how-to-contribute-signed-commits)=
 ### Signing commits
 
-All changes that go into the Kernel documentation repository require signed
-commits (using the `-S` or `--sign-off` option).
+All changes that go into the Kernel documentation repository need to be signed
+off (using the `-s` or `--signoff` option) by the contributor.
```

Comment:
removed all mentions of "signed commits" and only use "sign off" wording + options.